src/Workspaces/Core/Portable/SymbolSearch/SymbolSearchOptions.cs

```diff
@@ -4,18 +4,17 @@
 
 using System.Runtime.Serialization;
 
-namespace Microsoft.CodeAnalysis.SymbolSearch
+namespace Microsoft.CodeAnalysis.SymbolSearch;
+
+[DataContract]
+internal readonly record struct SymbolSearchOptions
 {
-    [DataContract]
-    internal readonly record struct SymbolSearchOptions(
-        [property: DataMember(Order = 0)] bool SearchReferenceAssemblies = true,
-        [property: DataMember(Order = 1)] bool SearchNuGetPackages = true)
-    {
-        public SymbolSearchOptions()
-            : this(SearchReferenceAssemblies: true)
-        {
-        }
+    [DataMember] public bool SearchReferenceAssemblies { get; init; } = true;
+    [DataMember] public bool SearchNuGetPackages { get; init; } = true;
 
-        public static readonly SymbolSearchOptions Default = new();
+    public SymbolSearchOptions()
```

Comment:
Yes, since this type is a struct. If we don't have explicit parameter-less ctor calling `new SymbolSearchOptions()` would not initialize the default values of the properties (it'd be equivalent to `default(SymbolSearchOptions)`).